app/src/main/report/components/MainMenu/MainMenu.scss

```diff
@@ -1,13 +0,0 @@
-.menu {
```

Comment:
these styles are obsolete, bootstrap has similar styles (and we're redesigning this page anyway)